Src/FluentAssertions/Events/EventMonitor.cs

```diff
@@ -19,18 +19,22 @@ internal class EventMonitor<T> : IMonitor<T>
     private readonly ConcurrentDictionary<string, EventRecorder> recorderMap =
         new ConcurrentDictionary<string, EventRecorder>();
 
-    public EventMonitor(object eventSource, Func<DateTime> utcNow)
+    public EventMonitor(object eventSource, EventMonitorOptions options)
     {
         Guard.ThrowIfArgumentIsNull(eventSource, nameof(eventSource), "Cannot monitor the events of a <null> object.");
+        Guard.ThrowIfArgumentIsNull(eventSource, nameof(eventSource), "Event monitor needs configuration.");
```

Comment:
Was this intentional? (2nd `Guard` is the same)

Reply:
```suggestion
        Guard.ThrowIfArgumentIsNull(options, nameof(options), "Event monitor needs configuration.");
```